blueocean-commons/src/main/java/io/jenkins/blueocean/commons/ServiceException.java

```diff
@@ -108,6 +108,10 @@ public UnauthorizedException(ErrorMessage errorMessage, Throwable throwable ) {
         }
     }
 
+    /**
+     * @deprecated use #BadRequestException
+     */
+    @Deprecated
```

Comment:
yes go ahead and delete if you like - or up to @vivek - he can press merge if wants.